### PR TITLE
Jetpack-mu-wpcom: Show Hosting > Overview

### PR DESCRIPTION
**Pull request by miksansegundo**
Related https://github.com/Automattic/dotcom-forge/issues/7066

## Proposed changes:

* Add submenu for Hosting > Overview

|BEFORE|AFTER|
|-|-|
|Screenshot 2567-06-06 at 18 24 56|Screenshot 2567-06-06 at 18 29 44|

### Other information:

- [X] Have you written new tests for your changes, if applicable?
- [X] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [X] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to `/wp-admin` and confirm you see Hosting > Overview

### CODE REVIEW

File: projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php [hunk 1]
```diff
@@ -254,3 +254,28 @@ function wpcom_show_admin_interface_notice() {
...
+ * This logic should be moved to jetpack-mu-wpcom once nav redesign is fully launched.
+ */
+function add_hosting_overview_menu() {
+	if ( ! wpcom_is_nav_redesign_enabled() ) {
+		return;
+	}
+
+	$domain = preg_replace( '#^https?://#', '', network_site_url() );
+
+	add_submenu_page(
```
Comment (fushar) [inline projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php]: We already have this place for adding hosting menus; could we also add it there for consistency and co-location? https://github.com/Automattic/jetpack/blob/d74c4fa3c2f623bae22c8e5e635e69dba5b441b5/projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php#L52

Comment (miksansegundo) [reply projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php]: Sure ✅

**Review by fushar — COMMENTED** [review]
The Overview menu should be the second one (as it was before), so that when the parent Hosting menu is clicked, it goes to My Home instead of to Site Management Panel.

image

**Review by fushar — DISMISSED** [review]

**Comment by miksansegundo** [comment]
I had to rebase and fix versions

**Review by taipeicoder — DISMISSED** [review]

**Comment by miksansegundo** [comment]
Thanks @taipeicoder but I have to fix versions again

**Review by fushar — APPROVED** [review]

**Pull request merged**